kontrakter/siktelseTiltale/1.0/siktelseTiltale.schema.json

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
@calle-777 :
ansattPerson.

Bør vi kreve mer data?
Om feltet brukes for å kontakte person, bør vi kreve 'kontaktdata'.
Avsender er jo valgfri. Så da bør den strammes inn. Om det er forenklet fordi det er brukt flere plasser bør det skilles ut i to objekter

Reply:
Avsender person påbudt, sammen med kontaktInfo epost som vi alltid kan love. Utestående kontaktinformasjon som distriktet ønsker domstolene skal bruke.